rt/frontend/jaxws/src/main/java/org/apache/cxf/jaxws/WrapperClassGenerator.java

```diff
@@ -109,7 +109,11 @@ private String getPackageName(Method method) {
         return list;
     }
 
-    public Set<Class<?>> generate() {
+    public Set<Class<?>> generate(JaxWsServiceFactoryBean factory, InterfaceInfo ii, boolean q) {
+        this.qualified = q;
```

Comment:
Please remove `qualified` and `interfaceInfo` from class members, pass them as arguments.

Reply:
I can not. The verify limit the number of parameters to seven max. Else I have to create an internal class context to forward it between method.

Reply:
You mean checkstyle?  
// CHECKSTYLE: OFF
....
// CHECKSTYLE: ON
(no very great but will do the job)